test/e2e/launch.js

```diff
@@ -232,6 +232,12 @@ async function writeLogs(app, location) {
 async function startApp(app, awaitingSelector = `.tm-session`) {
   console.log(`Starting app`)
   await app.start()
+  if (!app.browserWindow) {
+    console.log(`No browser window`)
+    return
```

Comment:
should this crash?

Reply:
It should. Do you want me to remove those lines ?